app/org/sagebionetworks/bridge/play/controllers/HealthDataController.java

```diff
@@ -31,6 +36,21 @@ final void setHealthDataService(HealthDataService healthDataService) {
         this.healthDataService = healthDataService;
     }
 
+    /** Gets a list of records for the given healthCode between the specified createdOn times (inclusive). */
+    public Result getRecordsByCreatedOn(String createdOnStartString, String createdOnEndString) throws IOException {
+        UserSession session = getAuthenticatedAndConsentedSession();
```

Comment:
This is an authenticated user API?

Reply:
As per IM conversation, I'm using a DateTimeRangeResourceList because this is intended for integ tests, and I didn't think the extra complexity of using a ForwardCursorPagedResourceList was worth it since no one was asking for it.

Also as per IM conversation, adding it as a user API is fine. Other alternatives considered are developer API or an internal admin API (but definitely not researcher API).